apps/chrome-devtools/src/extension/wrap.ts

```diff
@@ -13,33 +17,44 @@ window.postMessage({
   }
 } as OtterMessage<ConnectContentMessage, typeof applicationMessageTarget>, '*');
 
+declare namespace globalThis {
+  let localMessageListener: ((this: Window, ev: MessageEvent<any>) => any) | undefined;
+}
+
 // Remove previous listener (if the script is reloaded by a previous extension instance)
-if ((globalThis as any).localMessageListener) {
-  window.removeEventListener('message', (globalThis as any).localMessageListener);
+if (globalThis.localMessageListener) {
+  window.removeEventListener('message', globalThis.localMessageListener);
 }
 
 /**
  * Listener for messages from the page
  * @param event
  */
-// eslint-disable-next-line prefer-arrow/prefer-arrow-functions
-(globalThis as any).localMessageListener = function (event: MessageEvent<any>) {
+function messageListener(event: MessageEvent<any>) {
   let message: any;
+  /* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access -- message is of type `any` */
   try {
     message = typeof event.data === 'string' ? JSON.parse(event.data) : event.data;
   } catch (e) {
     // ignore invalid JSON
+    // eslint-disable-next-line no-console -- needed to warn the user
```

Comment:
could be disabled at file level, this is specific to this file that is injected into a page and cannot have additional dependencies